src/screens/SendTokens.js

```diff
@@ -211,13 +349,28 @@ class SendTokens extends React.Component {
       )
     }
 
+    const renderAlertBody = () => {
+      if (this.state.ledgerStep === 0) {
+        return (
+          <div>
+            <p>Please go to you ledger and validate each output of your transaction.</p>
+            <p>Press both buttons on your ledger is case the output is correct.</p>
```

Comment:
Suggestion: "Press both buttons on your ledger to confirm the transference."

Reply:
Changed a bit to reflect more the process. Now its:

```
Please go to you Ledger and validate each output of your transaction. Press both buttons in case the output is correct.

In the end, a final screen will ask you to confirm sending the transaction.
```

Reply:
Great. Thanks!